ppr-ui/src/utils/validation-helper.ts

```diff
@@ -347,6 +347,9 @@ export function isObjectEqual (object1: any, object2: any): boolean {
   //
   // Does an equality between 2 objects
   //
+  if (object1 === undefined) object1 = null
+  if (object2 === undefined) object2 = null
+  // these ifs as Object.create() needs null or defined object
```

Comment:
Looks good..

Reply:
Not really. There are other cleaner ways to do it. We can either have default values as arguments or when creating an Object: `isObjectEqual (object1: any = null, object2: any = null)` or `Object.create(object1 || null)`

I'm just unsure if we even need to check if objects are equal if one is undefined/null. We can simply do `if (!object1 || !object2) return false`.